Feature/cvpr reviewer recommendation

melisabok opened the pull request.

celestemartinez commented:
When I try to see the edge browser as an AC, I get an error because ACs are not readers of the `thecvf.com/CVPR/2023/Conference/Area_Chairs/-/Assignment` invitation, shouldn't they be?

melisabok commented:
> When I try to see the edge browser as an AC, I get an error because ACs are not readers of the thecvf.com/CVPR/2023/Conference/Area_Chairs/-/Assignment invitation, shouldn't they be?

Fixed. I fixed the readers in the live site.
